### ISSUE TITLE
pywinusb 0.4.2 and pyinstaller 3.3.1 / Python 2.7

### ISSUE THREAD

**Issue by fabianbuettner** [pyinstaller/pyinstaller#3902]
Hi,

I am trying to solve a problem with pyinstaller 3.3.1 (and also the developer version from today) and pywinusb 0.4.2 using python 2.7 on windows 10.

pywinusb does not seem to find any USB devices after being packaged by pyinstaller. Just calling the script via the python 2.7 interpreter works.

I attached a small sample script which I found in the [pywinusb issues](https://github.com/rene-aguirre/pywinusb/issues/45).
[test.zip](https://github.com/pyinstaller/pyinstaller/files/2644122/test.zip)

If I do the same using python 3.6, pywinusb 0.4.2 finds all USB devices even after being packaged by pyinstaller 3.3.1.

I have python 2.7 and python 3.6 installed in parallel via choco as 64-bit versions.

A smilliar [issue ](https://github.com/rene-aguirre/pywinusb/issues/45) was closed in the pywinusb repository by the guy reporting the issue can be worked around using the 32-bit Version of python 2.7.  But since the library is working fine using the 64-bit interpreter this does not make sense to me.

regards,
Fabian

**Comment by htgoebel** [comment]
This looks like not all required files are packaged when using Python 3.7. Please have a look at [Make sure everything is packaged correctly](https://github.com/pyinstaller/pyinstaller/wiki/How-to-Report-Bugs#make-sure-everything-is-packaged-correctly) for tips to find out what is missing. Build in `--one-dir` mode and compare which files are collected in the `dist` directory. Use the "pyi-archiveviewer" tool coming with PyInstaller to inspect the `.pyz`-file in the `build` directory and compare both.

**Comment by fabianbuettner** [comment]
Thanks for your answer.
I tried building in `--one-dir` mode already but it didn't help.
To be honest, I have no idea what I am currently looking for.
Output of pyinstaller from python 2.7 and the output of pyinstaller from python 3.6 looks quite different.
I will have to read more about pyinstaller and pyi-archiveviewer to be able to make use of this information.
[dist_test27_directory_listing.txt](https://github.com/pyinstaller/pyinstaller/files/2644415/dist_test27_directory_listing.txt)
[dist_test36_directory_listing.txt](https://github.com/pyinstaller/pyinstaller/files/2644416/dist_test36_directory_listing.txt)
[build_test_27_pyi-archive_viewer.txt](https://github.com/pyinstaller/pyinstaller/files/2644417/build_test_27_pyi-archive_viewer.txt)
[build_test_36_pyi-archive_viewer.txt](https://github.com/pyinstaller/pyinstaller/files/2644418/build_test_36_pyi-archive_viewer.txt)

**Comment by htgoebel** [comment]
Please check for differences your self.

**Comment by htgoebel** [comment]
Okay, I should be a bit more verbose: Since it is working when using Python 3.6 and not working when using Python 3.7, there msut be differences in whar files are packed. So the very first step is to search for these differences.

**Comment by fabianbuettner** [comment]
Please note, that I am talking about Python 2.7 and Python 3.6 and **not** 3.6 and 3.7!

And I had already written: 
> I will have to read more about pyinstaller and pyi-archiveviewer to be able to make use of this information.

**Comment by htgoebel** [comment]
> Please note, that I am talking about Python 2.7 and Python 3.6 and not 3.6 and 3.7!

IC. Anyway, this does not matter related to this case.

**Comment by bwoodsend** [comment]
Closing as this appears to by Python 2 only which is no longer supported. If it crops up again on modern versions of Python then please raise a new issue.
